src/nunit-gui/Views/MainForm.cs

```diff
@@ -163,5 +163,11 @@ protected override void OnDragDrop(DragEventArgs drgevent)
             if (files != null)
                 FilesDragAndDropped?.Invoke(files);
         }
+
+        protected override void OnDragEnter(DragEventArgs drgevent)
+        {
+            if (drgevent.Data.GetDataPresent(DataFormats.FileDrop))
+                drgevent.Effect = DragDropEffects.Copy;
+        }
```

Comment:
sorry I have a habbit of fixing things that I know are wrong and are easy to fix instead of leaving a note. Really this and the designer file can be moved to a different PR for a different day in the long run. Boy scout rule was kicking in on these.